vagrant/scripts/get-kubeconfig.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This script replaces kubeconfig with configuration for vagrant cluster
+
+readonly DIR="$(realpath "$(dirname "${BASH_SOURCE[0]}")")"
+readonly ROOT_DIR="${DIR}/../../"
```

Comment:
Wouldn't it be better to operate in `/tmp/` instead of littering the repo?

Reply:
It may be gitignored but still it adds "one-use" junk to repo dir.

Reply:
check it now